sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuSortExec.scala

```diff
@@ -113,7 +115,7 @@ case class GpuSortExec(
   private [this] lazy val targetSize = RapidsConf.GPU_BATCH_SIZE_BYTES.get(conf)
 
   override def doExecuteColumnar(): RDD[ColumnarBatch] = {
-    val sorter = new GpuSorter(sortOrder, output)
+    val sorter = new GpuSorter(gpuSortOrder, output)
```

Comment:
Could we have a follow on issue to look at what to do with GpuSorter if we know the CPU sort order too? I did some less than ideal things with materializing columns when the we had to do out of core sort for a GPU sort order that required computation. I am not sure if we can remove that or not, but it would be good to investigate.

Reply:
Filed #3731